src/epd_driver/font.c

```diff
@@ -151,6 +151,7 @@ static enum EpdDrawError IRAM_ATTR draw_char(const EpdFont *font, uint8_t *buffe
     int color_difference = (int)props->fg_color - (int)props->bg_color;
     color_lut[c] = max(0, min(15, props->bg_color + c * color_difference / 15));
   }
+  char background_needed = props->flags & EPD_DRAW_BACKGROUND;
```

Comment:
I'd use the bool type here for readability. Since the esp is a 32 bit system, there is no reason to prefer char.